samcli/local/docker/lambda_build_container.py

```diff
@@ -96,7 +98,8 @@ def _make_request(protocol_version,
                       manifest_file_name,
                       runtime,
                       optimizations,
-                      options):
+                      options,
```

Comment:
Just making sure, the protocol version is passed in by importing the builders library right?

Reply:
Correct.